README.md

```diff
@@ -3180,6 +3190,15 @@ cd $(mktemp -d)
 # for example, this will create a temporary directory "/tmp/tmp.TivmPLUXFT"
 ```
 
+##### Quickly return to last working directory
```

Comment:
we can use 
##### Return to the previous directory
```bash
cd -
```

i suggest removing the comments below as they are hard to understand:
# example that will get you back to home directory:
# cd ~
# cd /tmp/tmp.TivmPLUXFT
# cd -



as most oneliners are quick